test/gateway/core_utils.js

```diff
@@ -68,7 +52,7 @@ module.exports.checkOpenSTProvenEvent = (event, _blockHeight, _storageRoot, wasA
     assert.equal(event["wasAlreadyProved"], wasAlreadyProved);
 };
 
-module.exports.checkStateRootCommittedEvent = (event, _blockHeight, _stateRoot) => {
+module.exports.checkStateRootAvailableEvent = (event, _blockHeight, _stateRoot) => {
```

Comment:
L48: We are not using `checkOpenSTProvenEvent` anymore, we can delete it.

Reply:
🦅 removed